### ISSUE TITLE
Authentication broken?

### ISSUE THREAD

**Issue by asingletary** [AngellusMortis/sxm-client#4]
* sxm-client version: 0.2.8
* Python version: 3.8
* Operating System: Ubuntu 20.04

Authentication seems to be broken?

ERROR:/usr/local/lib/python3.8/dist-packages/sxm/client.py:An Exception occurred when trying to perform the POST request!
ERROR:/usr/local/lib/python3.8/dist-packages/sxm/client.py:Params: {'moduleList': {'modules': [{'moduleRequest': {'resultTemplate': 'web', 'deviceInfo': {'osVersion': 'Windows', 'platform': 'Web', 'sxmAppVersion': '5.36.514', 'browser': 'Firefox', 'browserVersion': '89.0', 'appRegion': 'US', 'deviceModel': 'EverestWebClient', 'clientDeviceId': 'null', 'player': 'html5', 'clientDeviceType': 'web'}, 'standardAuth': {'username': 'XXX', 'password': 'XXX'}}}]}}
ERROR:/usr/local/lib/python3.8/dist-packages/sxm/client.py:Method: POST
ERROR:/usr/local/lib/python3.8/dist-packages/sxm/client.py:Request: <Request('POST', 'https://player.siriusxm.com/rest/v2/experience/modules/modify/authentication')>
ERROR:/usr/local/lib/python3.8/dist-packages/sxm/client.py:An Exception occurred when trying to perform the POST request!
ERROR:/usr/local/lib/python3.8/dist-packages/sxm/client.py:Params: {'moduleList': {'modules': [{'moduleRequest': {'resultTemplate': 'web', 'deviceInfo': {'osVersion': 'Windows', 'platform': 'Web', 'sxmAppVersion': '5.36.514', 'browser': 'Firefox', 'browserVersion': '89.0', 'appRegion': 'US', 'deviceModel': 'EverestWebClient', 'clientDeviceId': 'null', 'player': 'html5', 'clientDeviceType': 'web'}, 'standardAuth': {'username': 'XXX', 'password': 'XXX'}}}]}}
ERROR:/usr/local/lib/python3.8/dist-packages/sxm/client.py:Method: POST
ERROR:/usr/local/lib/python3.8/dist-packages/sxm/client.py:Request: <Request('POST', 'https://player.siriusxm.com/rest/v2/experience/modules/modify/authentication')>

**Comment by Crumbles-Pieces** [comment]
I have the same issue, everything was working fine and it only started happening to me last week. Did you find a resolution for this?

**Comment by asingletary** [comment]
Nope, never resolved.  I haven't had any success since late-2021.

**Comment by DCH3416** [comment]
I also observe this issue on Ubuntu. Not on macOS

Edit: Works now on my Ubuntu system. I wonder if it's just a quirk of how SXM handles authentication

**Comment by timgray** [comment]
I also have this issue.   I notice there is no responses so I assume this project is dead and orphaned?

**Comment by DCH3416** [comment]
> I also have this issue. I notice there is no responses so I assume this project is dead and orphaned?

This happens every once in a while and appears to be a feature from SXM. There may not be much that can be done about this particular issue.
